tasks/disable_transparent_hugepages.yml

```diff
@@ -1,18 +1,16 @@
 ---
 - name: Create disable transparent hugepages systemd service file
-  copy:
+  template:
     src: "{{ item }}"
     dest: /etc/systemd/system/disable-transparent-hugepages.service
     owner: root
     group: root
     mode: 0644
   with_first_found:
-    - "disable-transparent-hugepages.{{ ansible_distribution | lower }}.service"
-    - "disable-transparent-hugepages.{{ ansible_os_family | lower }}.service"
+    - "disable-transparent-hugepages.{{ ansible_distribution | lower }}.service.j2"
```

Comment:
does mongodb recommend to disable it ?
I was reading the [doc](https://blog.nelhage.com/post/transparent-hugepages/#:~:text=%E2%80%9CTransparent%20Hugepages%E2%80%9D%20is%20a%20Linux,default%20in%20most%20Linux%20distributions.) to understand more about it and they say it may be bad for certain DBs but it also say we could set `madvise` so this change will allow applications that are optimized for transparent hugepages to obtain the performance benefits, and prevent the associated problems otherwise.

Reply:
mongodb explicitely recommends disabling huge pages